CODINGSTYLE.md

```diff
@@ -42,7 +42,11 @@
     - [Specifying Map Capacity Hints](#specifying-map-capacity-hints)
     - [Specifying Slice Capacity](#specifying-slice-capacity)
 
-## Consistent Spelling
+## Consistent Spelling & Naming
```

Comment:
So I think we need to update the table of contents with `Naming`.

Also, can we add a point here about adding new libraries? I think its better to be here than in contributing guide.

Reply:
Nice catch, fixed! As for the libraries, I wouldn't add them here as it has little to do with the coding style. CODING.md would be a more appropriate place.